pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -340,6 +429,9 @@ func (ctrl *PersistentVolumeController) syncUnboundClaim(claim *v1.PersistentVol
 		if err != nil {
 			return err
 		}
+		// add an entry for prebinding metric reporting, however defer to fill
+		// the operator name until a volume has been matched for binding
+		ctrl.addOperationTimestamp("prebinding", claimToClaimKey(claim), "")
```

Comment:
do we want to count time if the PV user requested is missing?  That seems like a user error condition.

Reply:
i prefer to only count system time and exclude time caused by user error

Reply:
just want to make sure I understand, you meant ONLY report the metric when found == true and the volume is either available or bound to the claim? I was thinking about the following situation:
user specify a volume ref when creating PVC where there exists no volume initially, then a PV is created manually, binding will happen in the following syncUnboundClaim calls. In this case, the end to end latency to user may actually reveal some of their flaws in procedure.

Reply:
no longer valid after the removal of binding/prebinding recording